depmapomics/config.py

```diff
@@ -1,7 +1,202 @@
+import numpy as np
+
+########################## GENERIC PARAMS
+
 CACHE_PATH = '~/.depmapomics/'
 TMP_PATH = '/tmp/'
 ENSEMBL_SERVER_V = "http://nov2020.archive.ensembl.org/biomart"
 
+SHEETCREDS = '../.credentials.json'
+MY_ID = '~/.client_secret.json',
+MYSTORAGE_ID = "~/.storage.json"
+
+SHEETNAME = 'ccle sample tracker'
+
+TAIGA_ETERNAL = 'depmap-a0ab'
+
+
+REFSHEET_URL = "https://docs.google.com/spreadsheets/d/1Pgb5fIClGnErEqzxpU7qqX6ULpGTDjvzWwDN8XUJKIY"
+
+PRIVACY_RELEASE = "https://docs.google.com/spreadsheets/d/115TUgA1t_mD32SnWAGpW9OKmJ2W5WYAOs3SuSdedpX4"
+DEPMAP_PV = "https://docs.google.com/spreadsheets/d/1uqCOos-T9EMQU7y2ZUw4Nm84opU5fIT1y7jet1vnScE"
+
+POTENTIAL_LIST = "https://docs.google.com/spreadsheets/d/1YuKEgZ1pFKRYzydvncQt9Y_BKToPlHP-oDB-0CAv3gE"
+
+depmap_taiga = "arxspan-cell-line-export-f808"
+
+SAMPLEID="DepMap_ID"
+
+SAMPLESETNAME = "21Q2"
+
+VIRTUAL_FOLDER = "8d9c4c0691154a1f86b1b6e67c3fb683"
+
+RELEASE = SAMPLESETNAME.lower()
+
+############## TERRA
+
+HG38BAMCOL = ['internal_bam_filepath',
+              "internal_bai_filepath"]
+
+############## LOADING
+
+MAXAGE = '2020-11-01'
+
+EXTRACT_TO_CHANGE = {'from_arxspan_id': 'participant'}
+
+REPLACE = {'T': 'Tumor', 'N': 'Normal', 'm': 'Unknown', 'L': 'Unknown'}
+
+values = ['legacy_bam_filepath', 'legacy_bai_filepath'],
+
+filetypes = ['bam', 'bai']
+   
+match=['ACH-', 'CDS-']
+
+## old GP storage buckets
+#rnaworkspace2 = "broad-firecloud-ccle/CCLE_DepMap_RNAseq"
+#rnaworkspace4 = "broad-genomics-delivery/Cancer_Cell_Line_Factory_CCLF_RNAseq"
+#rnaworkspace5 = "nci-mimoun-bi-org/CCLF_RNA_2_0"
+#rnaworkspace3 = "broad-genomics-delivery/CCLE_DepMap_RNAseq"
+#rnaworkspace1 = "broad-genomics-delivery/Getz_IBM_CellLines_RNASeqData"
+
+## curent GP buckets
+rnaworkspace6 = "terra-broad-cancer-prod/CCLE_DepMap_RNAseq"
```

Comment:
this might be a paint to refactor given that it's inside a lot of ipynbs, otherwise I vote for all capital naming for global variable.